ui/apps/platform/src/services/PolicyCategoriesService.ts

```diff
@@ -1,15 +1,8 @@
+import { PolicyCategory } from 'types/policy.proto';
 import axios from './instance';
 
 const policyCategoriesUrl = '/v1/policycategories';
 
-export type PolicyCategory = {
-    id: string;
-    // central/policycategory/service/service_impl.go
-    // policy category must have a name between 5 and 128 characters long with no new lines or dollar signs
-    name: string;
-    isDefault: boolean;
-};
-
```

Comment:
This is an incorrect change, but we can sort it out in the next step:

* This type (with comment that I found in go implementation code) corresponds to https://github.com/stackrox/stackrox/blob/master/proto/api/v1/policy_category_service.proto#L14-L18
* When a service defines a response type, which is the recent pattern in 2022, it is the backend source of truth, even if there is a corresponding storage type in https://github.com/stackrox/stackrox/blob/master/proto/storage/policy_category.proto#L11-L15
* There is no `PolicyCategory` type in https://github.com/stackrox/stackrox/blob/master/proto/storage/policy.proto